x-pack/plugins/uptime/server/lib/alerts/status_check.ts

```diff
@@ -81,16 +82,18 @@ export const generateFilterDSL = async (
 export const formatFilterString = async (
   dynamicSettings: DynamicSettings,
   callES: ESAPICaller,
+  callAsCurrentUser: ElasticsearchClient,
   filters: StatusCheckFilters,
   search: string,
   libs?: UMServerLibs
 ) =>
   await generateFilterDSL(
     () =>
       libs?.requests?.getIndexPattern
-        ? libs?.requests?.getIndexPattern({ callES, dynamicSettings })
+        ? libs?.requests?.getIndexPattern({ callES, callAsCurrentUser, dynamicSettings })
```

Comment:
`callAsCurrentUser` should probably be renamed to something like `esClient` but I'll allow the uptime owners to choose what they like best